R/bundle.R

```diff
@@ -456,6 +456,12 @@ inferAppMode <- function(appDir, appPrimaryDoc, files, quartoInfo) {
     if (!is.null(quartoInfo)) {
       return("quarto-static")
     } else {
+      # For Shinyapps and rstudio.cloud, treat "rmd-static" app mode as "rmd-shiny" so that
+      # they can be served from a shiny process in Connect until we have better support of
+      # rmarkdown static content
+      if (isCloudServer) {
+        return("rmd-shiny")
```

Comment:
Actually tested by running this locally and getting an `rmd-static` output to deploy successfully?

Reply:
Yes, tested and verified